### DISCUSSION TITLE
Building a connector destination in a different language (Golang)

### DISCUSSION THREAD

**Discussion by margaritagomez** [airbytehq/airbyte#34280] [Connector Questions] [answered]
Hi! I'm building a destination in Go. The Docker image manages to successfully run the `spec`, `check` and `write` commands, just like the Java example in the docs describes it does. 

The docs do mention that with the Docker image, any language should be ok, but I don't see any other connectors in languages other than Python or Java. 

Can I proceed with a PR with my Go project, its Dockerfile, and everything? Or would there be issues with the CI/CD pipeline?

**Comment by marcosmarxm** [comment, marked as answer]
@margaritagomez, the project only accepts sources built with the official CDK (Python and Java). While you can create sources or destinations in any language of your choice, the project cannot support or maintain them. It's your responsibility to manage them for your company project.
